temporal-testing-junit5/src/test/java/io/temporal/testing/TestWorkflowExtensionTest.java

```diff
@@ -0,0 +1,57 @@
+package io.temporal.testing;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowOptions;
+import io.temporal.worker.Worker;
+import io.temporal.workflow.Workflow;
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+import java.time.Duration;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.slf4j.Logger;
+
+public class TestWorkflowExtensionTest {
+
+  @RegisterExtension
```

Comment:
Are we able to combine both extensions together? If so, let's add a test that shows that and executes workflow that runs some activity.

Reply:
Yes, extensions can be combined. However, my understanding was that the `TestActivityEnvironment` is only useful to test activities in isolation, as `TestWorkflowEnvironment` can handle both workflows and activities on its own?

I can extend `TestWorkflowExtensionTest` to have an activity in addition to workflow to clarify that.

Reply:
Please do, otherwise it's looking good to me.

Reply:
I've created PR https://github.com/temporalio/sdk-java/pull/340 to extend this test.